rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @book.rentals << self
+
+    @person = person
+    @book.rentals << self
```

Comment:
- [ ] The rental should be added to the array of the `person`, not `@book` again.

  ```
  person.rentals << self
  ```